Casks/zenbeats.rb

```diff
@@ -0,0 +1,22 @@
+cask 'zenbeats' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://static.roland.com/assets/media/pkg/Zenbeats_Installer.pkg'
+  name 'Zenbeats'
+  homepage 'https://www.roland.com/us/products/zenbeats/'
+
+  pkg 'Zenbeats_Installer.pkg'
+
+  uninstall pkgutil: 'jp.co.roland.zenbeats.Zenbeats.pkg',
+            quit:    'jp.co.roland.zenbeats',
+            signal:  [
+                       ['KILL', 'jp.co.roland.zenbeats'],
+                     ]
+
+  zap trash: [
+               '~/Library/Caches/com.juce.locks/juceAppLock_Roland Zenbeats',
+               '~/Library/Caches/jp.co.roland.zenbeats',
+               '~/Library/Saved Application State/jp.co.roland.zenbeats.savedState',
```

Comment:
You should also delete the shared library

```suggestion
               '/Users/Shared/Zenbeats Library',
               '~/Library/Caches/com.juce.locks/juceAppLock_Roland Zenbeats',
               '~/Library/Caches/jp.co.roland.zenbeats',
               '~/Library/Saved Application State/jp.co.roland.zenbeats.savedState',
```

Reply:
I’d rather not because it’s where user data goes.

Reply:
That is true but `zap` is never performed automatically and it is specifically intended to remove all traces of an app except files created by the user. ([docs](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md))
As far as I understand the library contains downloaded samples from Roland's repository and not user-created files?
![image](https://user-images.githubusercontent.com/1170440/78913674-a4ab0100-7a89-11ea-9386-27bbabd3a057.png)

Reply:
I don’t know. The library has subdirectories like `Songs` and `Tracks` so I figured I’d err on the side of caution.

Reply:
There’s [some limited evidence](https://amchannel.info/C0kz2bS_2xvtoEdXH-OUoKw) that people seem to be aware of the `Library` directory. Looks like they keep some content there that may not be available for re-download if `zap`ped.

Reply:
Well, in that case, ignore it 👍